include/solvers/heat_transfer.h

```diff
@@ -125,38 +159,41 @@ class HeatTransfer : public AuxiliaryPhysics<dim, GlobalVectorType>
    * @brief Call for the assembly of the matrix and the right-hand side.
    *
    * @deprecated This function is to be deprecated when the new assembly mechanism
-   * is integrated to this solver
+   * is integrated to this solver.
    */
   void
   assemble_matrix_and_rhs();
 
   /**
-   * @brief Call for the assembly of the right-hand side
+   * @brief Call for the assembly of the right-hand side.
    *
    * @deprecated This function is to be deprecated when the new assembly mechanism
-   * is integrated to this solver
+   * is integrated to this solver.
    */
   void
   assemble_rhs();
 
   /**
    * @brief Call for the assembly of the matrix and the right-hand side of the Nitsche restriction for the heat transfert equation.
    *
-   * @param assemble_matrix boolean that is true for matrix assembly, and false for rhs assembly
+   * @param assemble_matrix boolean that is true for matrix assembly, and false for rhs assembly.
    */
   void
   assemble_nitsche_heat_restriction(bool assemble_matrix);
```

Comment:
```suggestion
  assemble_nitsche_heat_restriction(const bool assemble_matrix);
```
Good opportunity to make this a const :)